scripts/upgrade/autobrr.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# autobrr upgrader
+# ludviglundgren 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+if [[ ! -f /install/.autobrr.lock ]]; then
+    echo_error "autobrr doesn't appear to be installed!"
+    exit 1
+fi
+
+_restart_autobrr() {
+    for user in "${users[@]}"; do
+        # restart autobrr
+        systemctl try-restart "autobrr@${user}"
+    done
+    echo_progress_done "Service restarted"
+}
+
+autobrr_download_latest
```

Comment:
Function missing here
```suggestion
#shellcheck source=sources/functions/autobrr
. /etc/swizzin/sources/functions/autobrr
autobrr_download_latest
```